src/Orchestration/Orchestration.php

```diff
@@ -19,6 +20,57 @@ public function __construct(Adapter $adapter)
         $this->adapter = $adapter;
     }
 
+    /**
+     * Command Line String into Array
+     *
+     * @param string $command
+     *
+     * @return (false|string)[]
+     */
+    public function parseCommandString(string $command): array {
```

Comment:
How is this command used? We should probably add a lot more documentation with it, seems a bit complex and hard for me to understand its responsibility with a quick glance.